unidist/core/backends/mpi/core/communication.py

```diff
@@ -311,20 +394,22 @@ def mpi_recv_object(comm, source_rank):
     return comm.recv(source=source_rank, tag=common.MPITag.OBJECT)
 
 
-def mpi_send_buffer(comm, buffer_size, buffer, dest_rank):
+def mpi_send_buffer(comm, buffer, dest_rank, data_type=MPI.CHAR, buffer_size=None):
```

Comment:
This method hasn't been used anywhere, but it's still there because we have 2 sending options (sync and isync) that can be used in the future, and data_size is a required argument.